cdec/update-server-packaging-war/src/main/java/com/codenvy/im/update/ReportService.java

```diff
@@ -47,18 +51,29 @@ public ReportService() {
     @GET
     @Path("/parameters/{report_type}")
```

Comment:
What is bad? 
Report parameters isn't private info. 
This is Update Server, and we have a few public methods as well there.